partials/settings.html

```diff
@@ -15,15 +15,15 @@
       <ol>
         <li>
           <input name="title" placeholder="Server address (http://...)" 
-            dir="auto" type="text" ng-model="credentials.address">
+            dir="auto" type="text" ng-model="credentials.address"/>
         </li>
         <li>
           <input name="title" placeholder="Username" dir="auto" type="text"
-            ng-model="credentials.username">
+            ng-model="credentials.username"/>
```

Comment:
`input` is a void element in HTML5, it doesn't need to be closed, see http://www.w3.org/html/wg/drafts/html/master/syntax.html#void-elements